app/forms/waste_carriers_engine/cards_form.rb

```diff
@@ -9,15 +9,18 @@ def initialize(transient_registration)
 
     def submit(params)
       # Assign the params for validation and pass them to the BaseForm method for updating
-      self.temp_cards = params[:temp_cards]
+      self.temp_cards = if params[:temp_cards].present?
+                          params[:temp_cards]
+                        else
+                          0
+                        end
       attributes = { temp_cards: temp_cards }
 
       super(attributes, params[:reg_identifier])
     end
 
     # Must be a positive integer or 0
     # Leaving it blank is also valid - this will automatically sub in a 0 since the field is marked as an Integer
```

Comment:
I'd suggest updating the comment as well as we now know this is not true.

Reply:
Oops, good point! Will fix